woocommerce/woocommerce-gutenberg-products-block
# Add `FormStep` to Storybook

## What

Adds a storybook entry for the `FormStep` component we recently exported.

## Why

To showcase the component and provide a level of documentation.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Run `npm run storybook`
2. Open `localhost:6006` and check the `WooCommerce Blocks/@woocommerce-blocks-components/FormStep` component.
3. Try changing the controls and make sure it reflects in the canvas. (Note `stepHeadingContent` is a React Node so can't be changed in Storybook.)


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Skipping.
